services/galaxytoolshed/galaxytoolshed-activity.service.js

```diff
@@ -0,0 +1,41 @@
+import { age } from '../color-formatters.js'
+import { formatDate } from '../text-formatters.js'
+import BaseGalaxyToolshedService from './galaxytoolshed-base.js'
+
+export default class GalaxyToolshedCreatedDate extends BaseGalaxyToolshedService {
+  static category = 'activity'
+  static route = {
+    base: 'galaxytoolshed/created-date',
+    pattern: ':repository/:owner',
+  }
+
+  static examples = [
+    {
+      title: 'Galaxy Toolshed (created time)',
```

Comment:
Apologies bit of a nit but since we're using 'date' everywhere else let's do the same here:

```suggestion
      title: 'Galaxy Toolshed (created date)',
```

Reply:
Done in 353338867